src/website/src/app/documentation/demos/datagrid/detail/detail.html

```diff
@@ -0,0 +1,117 @@
+<!--
+  ~ Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+  ~ This software is released under MIT license.
+  ~ The full license information can be found in LICENSE in the root directory of this project.
+  -->
+
+<h2>Detail Pane</h2>
+
+<p>
+    When you need to show additional details for a record, the Detail Pane is a good pattern that will condense the datagrid to show a primary column and a panel to the right that you can display more details about your record. It is a great pattern to show the full content of the record in a larger space that can also scroll. The Detail Pane is also fully accessible for keyboard and screen reader users.
+</p>
+<p>
+    Adding the Detail Pane will add a new toggle icon in the left hand side of the Datagrid. When the pane is opened, it will take up 2/3 of the width of the Datagrid and automatically hide all columns, except for the first column and any built-in columns that facilitate features like selection and row actions. The pagination is also changed to a condensed format that scales well to small sizes. Only one row can be opened at a time, so selecting another row will change the content to the newly selected row.
+</p>
+<p>
+    The Detail Pane cannot be used at the same time as Expandable Rows, and if both are enabled at the same time the Detail Pane will take presedence. Also, hide and show columns is disabled while the Detail Pane is opened, but stil works properly when closed. The rest of the Datagrid behaviors should work as expected even while the Detail Pane is opened. 
```

Comment:
Minor spelling, active voice and present tense 

"The Detail Pane is not compatible with Expandable Rows; when both are enabled, the Detail Pane takes precedence. Hide and show columns are disabled while the Detail Pane is open, but still works properly when closed. The rest of the Datagrid behaviors work as expected, even while the Detail Pane is open."

Reply:
fixed